VirtualEconomyFramework/VENFTApp-Blazor/wwwroot/jsInterop.js

```diff
@@ -1,15 +1,35 @@
-﻿var barcodereader = null;
-var dotnetRef = null;
-var loadedImage = null;
-var html5QrCode = null;
-var ipfsHolder = null;
-var uploadReference = null;
-var ipfs = null;
-(async () => {
-    //global init
-})();
-
-window.setMusicInfo = (title, artist, description) => {
+﻿var barcodereader = null;
+var dotnetRef = null;
+var loadedImage = null;
+var html5QrCode = null;
+var ipfsHolder = null;
+var uploadReference = null;
+var ipfs = null;
+(async () => {
+    //global init
+})();
+
+var openedModals = [];
+function handleModalClose(element) {
+    const index = openedModals.indexOf(element);
+    if (index !== -1) {
+        openedModals.splice(index, 1);
+        if (openedModals.length == 0)
+            document.body.removeAttribute("style");
+    }
+}
+function handleModalOpen(element) {
+    if (openedModals.length === 0) {
+        const strollbarWidth = window.innerWidth - document.body.clientWidth;
+        const style = `width: calc(100% - ${strollbarWidth}px); overflow: hidden;`;
+        document.body.setAttribute("style", style)
+    }
+    const index = openedModals.indexOf(element);
+    if (index === -1)
+        openedModals.push(element);
+}

```

Comment:
This is to hide body scrollbar when some modal is opened. I can extend this to also hide parent modal scrollbar